index.jsx

```diff
@@ -169,14 +172,15 @@ export default function Dimensions ({
 
       render () {
         const {containerWidth, containerHeight} = this.state
-        if (this._parent && !containerWidth && !containerHeight) {
-          // only trigger a warning about the wrapper div if we already have a reference to it
+        const renderComponent = containerWidth || containerHeight || alwaysRender
```

Comment:
Don't we still need to care about `this._parent`?

Reply:
Would prefer `shouldRender` naming

Reply:
I dunno, if you don't like `alwaysRender`, how about `forceRender`? To me, if `shouldRender` is false, it means to suppress rendering, not to allow optimization. Another name could be `suppressOptimization`.

Reply:
I'm OK with `alwaysRender`, but `renderComponent` -> `shouldRender`